substrate/frame/alliance/src/benchmarking.rs

```diff
@@ -726,58 +765,71 @@ benchmarks_instance_pallet! {
 
 		let fellow2_lookup = T::Lookup::unlookup(fellow2.clone());
 		let call = Call::<T, I>::kick_member { who: fellow2_lookup };
-		let origin =
-			T::MembershipManager::try_successful_origin().map_err(|_| BenchmarkError::Weightless)?;
-	}: { call.dispatch_bypass_filter(origin)? }
-	verify {
+		let origin = T::MembershipManager::try_successful_origin()
+			.map_err(|_| BenchmarkError::Weightless)?;
+
+		#[block]
+		{
+			call.dispatch_bypass_filter(origin)?;
+		}
+
 		assert!(!Alliance::<T, I>::is_member(&fellow2));
 		assert_eq!(DepositOf::<T, I>::get(&fellow2), None);
-		assert_last_event::<T, I>(Event::MemberKicked {
-			member: fellow2,
-			slashed: Some(T::AllyDeposit::get())
-		}.into());
+		assert_last_event::<T, I>(
+			Event::MemberKicked { member: fellow2, slashed: Some(T::AllyDeposit::get()) }.into(),
+		);
+		Ok(())
 	}
 
-	add_unscrupulous_items {
```

Comment:
It looks like `add_unscrupulous_items` was forgotten. Could you please re-add it @0xmovses?

Reply:
Ah, yes sure thing.

Reply:
@ggwpez It's there, see here on master https://github.com/paritytech/polkadot-sdk/blob/7b06e634fe749508b639e50cccbeb58ce794323c/substrate/frame/alliance/src/benchmarking.rs#L785

Reply:
`add_unscrupulous_items` != `add_scrupulous_items`

Reply:
adding it back now.

Reply:
@ggwpez so it seems that there was never an `add_scrupulous_items`, but only a `add_unscruplous_items`, see master branch before this PR was merged https://github.com/paritytech/polkadot-sdk/blob/cfb29254f74412cea35e8048d8aea94bc789fcb1/substrate/frame/alliance/src/benchmarking.rs#L741
 in this PR, it seems the name of the function was wrongly changed. 
  
 Put up a fix here https://github.com/paritytech/polkadot-sdk/pull/2173